### PR TITLE
Adds more sanity to vending machines

### LINKED ISSUE #10470: Ghosts make shocked vending machines spark
(WEB REPORT BY: sprok REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> 5e515d87df38f5c8a7ef1f6269b28ee975d02535 
> 
> General description of the issue
> Clicking on electrified vending machine on the vox outpost makes sparks
> 
> What you expected to happen
> Open the menu to see what was in the machine
> 
> What actually happened
> That, and also sparks
> 
> Steps to reproduce if possible
> Electrify an outpost vending machine, click it as a ghost

### PR DESCRIPTION
**Pull request by JustSumBody**
Closes #10470 
Ghosts can no longer spark electrocuted vending machines, you can no longer shock yourself beyond death with the cut wire menu if you're already on the ground.

### CODE REVIEW

File: code/datums/wires/vending.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@ var/const/VENDING_WIRE_IDSCAN = 8
 
 /datum/wires/vending/CanUse(var/mob/living/L)
 	var/obj/machinery/vending/V = holder
-	if(!istype(L, /mob/living/silicon))
+	if(!istype(L, /mob/living/silicon) && !L.lying && !L.incapacitated())
```
Comment (ComicIronic) [inline code/datums/wires/vending.dm]: Does this not mean I can just rest to not be shocked? You probably want to check user.stat for dead or dying people.

Comment (9600bauds) [inline code/datums/wires/vending.dm]: that's what incapacitated() does

Comment (JustSumBody) [inline code/datums/wires/vending.dm]: Oh shit, you can avoid shocks if you're resting and the wirecutters are glued to your hands though.

Comment (9600bauds) [inline code/datums/wires/vending.dm]: Wouldn't you just be unable to use the wire tough?

Comment (JustSumBody) [inline code/datums/wires/vending.dm]: Nope, it's somehow working in my test server.

Comment (JustSumBody) [inline code/datums/wires/vending.dm]: Oh, the if statement after needs sanity too I think whoops

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -577,12 +577,13 @@ var/global/num_vending_terminals = 1
...
 	if(stat & (BROKEN|NOPOWER))
 		return
-
-	if(seconds_electrified > 0)
-		if(shock(user, 100))
-			return
-	else if (seconds_electrified)
-		seconds_electrified = 0
+		
+	if(!user.lying && !user.incapacitated())
```
Comment (9600bauds) [inline code/game/machinery/vending.dm]: This should be plain old returning if you're incapacitated

Comment (JustSumBody) [inline code/game/machinery/vending.dm]: The issue of not being able to see the vending machine's inventory as a ghost pops up though. 
As it is, ghosts can interact with it to see it but can't vend or spark it. 
~~it just werks~~
~~actually I'm an idiot~~
Actually yeah it's still somehow working properly how is this working

**Comment by 9600bauds** [comment]
:ok_hand:

**Pull request merged**